cmd/lncli/arg_parse.go

```diff
@@ -21,7 +21,7 @@ var secondsPer = map[string]int64{
 	"y": 31557600, // 365.25 days
 }
 
-// parseTime parses UNIX timestamps or short timeranges inspired by sytemd (when starting with "-"),
+// parseTime parses UNIX timestamps or short timeranges inspired by systemd (when starting with "-"),
```

Comment:
Can we break this before 80 chars if we're touching this line?

Reply:
Fixed.